enterprise-suite/console_test/util/util.go

```diff
@@ -0,0 +1,189 @@
+package util
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+	"time"
+)
+
+const DefaultTimeout = 3 * time.Second
+
+type CmdBuilder struct {
+	name                 string
+	args                 []string
+	timeout              time.Duration
+	expectZeroExitStatus bool
+	panicOnError         bool
+	printStdout          bool
+	printStderr          bool
```

Comment:
Shouldn't we default these to `true`? As I believe ginkgo will hide stdout/stderr unless the test fails, so no harm in logging all output.

Reply:
I tried it out, ginkgo prints out stdout/stderr by default on my machine. Maybe it's only in BeforeSuite/AfterSuite? Anyway, it gets quite noisy, I would vote for keeping the default `false`.